protos/telemetry/telemetry.proto

```diff
@@ -577,7 +577,11 @@ enum FixType {
 message Battery {
     uint32 id = 3 [(mavsdk.options.default_value)="0"]; // Battery ID, for systems with multiple batteries
     float voltage_v = 1 [(mavsdk.options.default_value)="NaN"]; // Voltage in volts
-    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0.0 to 1.0)
+    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0 to 100)
+    float current_battery_a = 4 [(mavsdk.options.default_value)="NaN"]; // Battery current in Amps, NAN if autopilot does not measure the current
+    float current_consumed_ah = 5 [(mavsdk.options.default_value)="NaN"]; // Consumed charge in Amp hours, NAN if autopilot does not provide consumption estimate
+    float energy_consumed_wh = 6 [(mavsdk.options.default_value)="NaN"]; // Consumed energy in Watt hours, NAN if autopilot does not provide energy consumption estimate
```

Comment:
From a MAVLink point of view this duplicates the field above (we don't send both, and mAh is a much more common representation in drones). If MAVSDK chooses to provide the conversion to make it easier for consumers I have no problem with it.

Reply:
It's not the same though. Amp hours is the Amps drawn over time. If you also have the voltage, then you can get to energy consumed.

The MAVLink message seems to have both actually:
https://mavlink.io/en/messages/common.html#BATTERY_STATUS

Reply:
@julianoes No one uses that in drones, and it is not in the new MAVLink message. But if you can derive it from the data that is provided, then sure.

Docs not live yet, but it is in development.xml here: https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/development.xml#L410-L425